components/CardList.tsx

```diff
@@ -0,0 +1,48 @@
+import Card from "./Card";
+import styled from "styled-components";
+import LinkNotFound from "./LinkNotFound";
+import { IPFolderData, IPCardData, ITransformCardData } from "../utils/type";
+
+interface Props {
+  cardData: ITransformCardData[];
+  folderListData?: IPFolderData[];
+  linkData?: IPCardData[];
+}
+
+export default function CardList({ cardData, folderListData, linkData }: Props) {
+  return (
+    <>
+      {cardData.length === 0 ? (
+        <LinkNotFound />
+      ) : (
+        <StyledCardListContainer>
+          {cardData.map((data: ITransformCardData) => (

```

Comment:
```suggestion
          {cardData.map((data) => (
```
이미 cardData의 타입이 명확하기 때문에 map의 콜백 인자를 굳이 타이핑 할 필요는 없습니다.
알아서 추론이 되어요.